### PR TITLE
Fix #580: updating package.json, package.json in src/frontend and gatsby-config.js

### LINKED ISSUE #580: Update gatsby-config.js file
**What would you like to be added**:
The `src/frontend/gastby-config.js` file still contains some boilerplate information that needs to be updated or removed to reflect our code.  See #541 for more information.

### LINKED ISSUE #581: Remove redundant information from frontend package.json
**What would you like to be added**:
The frontend's package.json file contains information that is either redundant or unnecessary and should be removed.  
This includes:
- [ ] remove prettier form devDependencies
- [ ] remove scripts associated with prettier
- [ ] remove `repository` key and associated value
- [ ] remove `bugs` and `url` keys and values
- [ ] anything else that looks like it should be in the project's root package.json

### PR DESCRIPTION
**Pull request by lozinska**
## Issue This PR Addresses
Fixes #580 , Fixes #581 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
Updating package.json, package.json and gatsby-config.json in src/frontend/. Removed redundant information. Shifting devDependencies from `src/frontend/package.json`
`"json-loader": "^0.5.7",
    "node-sass": "^4.13.1",
    "sass": "^1.24.4",`
to `package.json`

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/frontend/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "gatsby-starter-default",
+  "name": "telescope",
   "private": true,
-  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "description": "A tool for tracking blogs in orbit around Seneca's open source involvement",
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```
Comment (humphd) [inline src/frontend/package.json]: You can remove this.

File: src/frontend/package.json [hunk 2]
```diff
@@ -21,28 +21,15 @@
     "react-dom": "^16.12.0",
     "react-helmet": "^5.2.1"
   },
-  "devDependencies": {
```
Comment (humphd) [inline src/frontend/package.json]: You can revert this change, because @miggs125 is doing it in #647

File: package.json [hunk 3]
```diff
@@ -101,6 +101,9 @@
     "spam-filter": "1.1.1",
     "stoppable": "1.1.0",
     "valid-url": "1.0.9",
+    "json-loader": "^0.5.7",
```
Comment (humphd) [inline package.json]: You can revert this change, since we'll get it when we do our post-install.

File: src/frontend/gatsby-config.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 module.exports = {
   siteMetadata: {
-    title: `Gatsby Default Starter`,
-    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
-    author: `@gatsbyjs`,
+    title: `Telescope`,
+    description: `A tool for tracking blogs in orbit around Seneca's open source involvement`,
+    author: `ICT Students and professors`,
```
Comment (humphd) [inline src/frontend/gatsby-config.js]: Technically, I think we're called SDDS not ICT now :(

**Comment by humphd** [comment]
This can get rebased to pick up some changes in the frontend package.json.

File: src/frontend/package.json [hunk 5]
```diff
@@ -31,8 +27,8 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "start": "npm run develop",
     "serve": "gatsby serve",
+<<<<<<< HEAD
```
Comment (humphd) [inline src/frontend/package.json]: You have failed merge conflict markers in here causing the JSON parser to fail.

Comment (lozinska) [reply src/frontend/package.json]: Yeah, I see. Fixing it now

Comment (lozinska) [reply src/frontend/package.json]: @humphd I have updated my files but now I'm getting new error
image
I don't know what to do to fix this error

File: package.json (hunk 3 above)
Comment (humphd) [reply package.json]:
```
git checkout master package.json
```

And you can get rid of this change, which we don't need.

File: src/frontend/package.json [hunk 6]
```diff
@@ -31,15 +27,8 @@
...
-    "clean": "gatsby clean"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
-  },
-  "bugs": {
-    "url": "https://github.com/gatsbyjs/gatsby/issues"
+    "clean": "gatsby clean",
+    "compile-sass": "node-sass src/sass/main.scss src/css/style.css -w"
```
Comment (humphd) [inline src/frontend/package.json]: I think @miggs125 removed this line when he removed SASS.  It's safe to remove it here.

Comment (lozinska) [reply src/frontend/package.json]: @humphd line 32?

Comment (humphd) [reply src/frontend/package.json]: Yes, just line 32, sorry for not being clear.  Line 31 is good, leave it in.
